presto-common/src/main/java/com/facebook/presto/common/block/VariableWidthBlock.java

```diff
@@ -40,6 +40,12 @@
 import static io.airlift.slice.Slices.EMPTY_SLICE;
 import static java.lang.String.format;
 
+/**
+ * A block that contains positionCount slices (blobs).
```

Comment:
technically, the block is one slice representing postitionCount elements.  The offset within the slice of each element is defined in the offsets array.

Reply:
done